### DISCUSSION TITLE
Wishlist: Better Searching

### DISCUSSION THREAD

**Discussion by jlitrell** [strawberrymusicplayer/strawberry#741] [Ideas]
I think the one remaining issue from Amarok (other than Shout/Icecast streaming) is in searching a collection - when we argue about the album of the year, it would be great to be able to put in something like "year: 2021" to the search bar and have that populate.  (Genre etc. too, but that's less of an issue.)

**Comment by jonaski** [comment]
Works when testing here, the search has been rewritten since your post.
